bazel/coverage/collect_cc_coverage.sh

```diff
@@ -134,12 +136,12 @@ function gcov_coverage() {
           # Don't generate branch coverage (-b) because of a gcov issue that
           # segfaults when both -i and -b are used (see
           # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=84879).
-          "${GCOV}" -i $COVERAGE_GCOV_OPTIONS -o "$(dirname ${gcda})" "${gcda}"
+          "${GCOV}" -i "$COVERAGE_GCOV_OPTIONS" -o "$(dirname "${gcda}")" "${gcda}"
```

Comment:
im wondering if this should be an array, but it doesnt appear to be defined anywhere

Reply:
It's noted in the comment at the top as a var you set to pass extra options. I expect that also means it shouldn't get quoted or you won't be able to pass multiple options?

Reply:
> I expect that also means it shouldn't get quoted or you won't be able to pass multiple options?

yep - well that, or read into array (and make shellcheck happy)